controller/api/destination/server.go

```diff
@@ -62,7 +62,7 @@ func NewServer(
 	s := prometheus.NewGrpcServer()
 
 	// this server satisfies 2 gRPC interfaces:
-	// 1) linkerd2-proxy-api/destination.Destination (proxy-facing)
+	// 1) linkerd2-destination/destination.Destination (proxy-facing)
```

Comment:
Maybe this is an inadvertent rename? Am pretty sure "linkerd2-proxy-api" refers to the name of the repo here, which isn't changing.